content/concepts/did-ddo.md

```diff
@@ -369,7 +372,100 @@ Example:
   ]
 }
 ```
+ 
+#### Consumer Parameters
+
+Sometimes, you may need some input before consuming a dataset or running an algorithm.
```

Comment:
We do not use "consuming" anymore as no normal person speaks like that. Please use "download" or "accessing"